modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,210 @@
+[[content_langs]]
```

Comment:
Missing documentation for `typography_langs` and `typography_default_lang`. Note that `content_langs` already has a [dedicated entry in the docs](https://github.com/tinymce/tinymce-docs/blob/86adeabb818e5270cc2ce19f5dfd31ff67ed30e8/modules/ROOT/partials/configuration/content_langs.adoc)

Reply:
`content_langs` can influence the typography plugin by marking some sections of text with lang attribute. This attribute will be taken into account when typography plugin is about to apply language-specific rules. Not sure if that was the intention here